modules/gmc-agent/src/main/java/org/gridgain/service/sender/RetryableSender.java

```diff
@@ -22,6 +22,7 @@
 import java.util.concurrent.BlockingQueue;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import com.google.common.collect.Lists;
```

Comment:
Do we really need guava ?

Reply:
Nope, change on Collections.singletonList.